fleet_management_api/api_impl/controllers/stop.py

```
@@ -9,10 +9,10 @@
     text_response as _text_response,
 )
 from fleet_management_api.api_impl.api_logging import (
-    log_error_and_respond as _log_error_and_respond,
+    log_warning_or_error_and_respond as _log_warning_or_error_and_respond,
     log_info_and_respond as _log_info_and_respond,
     log_info as _log_info,
-    log_nok_and_respond as _log_nok_and_respond,
+    log_warning_or_error_and_respond as _log_warning_or_error_and_respond,
```

Ruff (0.8.2)

Redefinition of unused `_log_warning_or_error_and_respond` from line 12

Remove definition: `_log_warning_or_error_and_respond`

(F811)